src/sentry/api/event_search.py

```diff
@@ -2229,6 +2229,14 @@ def result_type_fn(function_arguments, parameter_values):
         ),
     ]
 }
+# In Performance TPM is used as an alias to EPM
+FUNCTION_ALIASES = {
+    "tpm": "epm",
+    "tps": "eps",
+}
+for alias, name in FUNCTION_ALIASES.items():
+    FUNCTIONS[alias] = deepcopy(FUNCTIONS[name])
+    FUNCTIONS[alias].name = alias
```

Comment:
Would be good to have a comment in `discover/fields.tsx` why these functions don't exist there.